_docs/_latest/use/cli/scanner.md

```diff
@@ -31,10 +31,17 @@ forseti model use <YOUR_MODEL_NAME>
 
 ### Running the scanner
 
+To run normal scanners:
```

Comment:
To run scanners (excluding external project access scanner):

Reply:
done